File: PHP/Exercicio010.php
<!DOCTYPE html>
<html>

<body>
 <h4>Qual o tipo de triângulo</h4>
 <form method="post" action="Exercicio010.php">
    <legend>Informe o 1o valor</legend>
    <input type="text" name="C1" id="C1">
    <legend>Informe o 2o valor</legend>
    <input type="text" name="C2" id="C2">
    <legend>Informe o 3o valor</legend>
    <input type="text" name="C3" id="C3">
    <button>Verificar</button>
 </form>
<?php
  if(isset($_POST['C1']) && 
     isset($_POST['C2']) &&
     isset($_POST['C3'])){
      $C1 = $_POST['C1'];
      $C2 = $_POST['C2'];
      $C3 = $_POST['C3'];

      if(($C1 + $C2) > $C3 && ($C1 + $C3 > $C2) && ($C2 + $C3 > $C1)){
        echo "<strong>Os tres lados formam um triangulo</strong>";
      }else{
          echo "<strong>Os tres lados nao formam um triangulo</strong>";
      }
  }
?>
</body>
</html>

File: JAVA/Exercicio010.java
package main;

import javax.swing.JOptionPane;

public class Exercicio010 {
	
	public static void main(String[] args) {
		int C1, C2, C3;
		C1 = Integer.parseInt(JOptionPane.showInputDialog("Informe o 1o comprimento:"));
		C2 = Integer.parseInt(JOptionPane.showInputDialog("Informe o 2o comprimento:"));
		C3 = Integer.parseInt(JOptionPane.showInputDialog("Informe o 3o comprimento:"));
		
		if(((C1 + C2) > C3) & ((C1 + C3) > C2) & ((C2 + C3) > C1)){
			JOptionPane.showMessageDialog(null, "Os três lados formam um triângulo");
		}else {
			JOptionPane.showMessageDialog(null, "Os três lados não formam um triângulo");
		}
	}

}
